react/components/ProductImages/styles.css

```diff
@@ -14,7 +14,7 @@
 .loader {
   height: -moz-available;
   height: -webkit-fill-available;
-  height: fill-available;
+  height: stretch;
```

Comment:
Did you test it on IE?

Reply:
Why use this unusual height attribute? A think it's better change to something else, like a flexbox.
This attr it's not safe to use.